### PR TITLE
Blog post about Mutiny variant of Vert.x API

### PR DESCRIPTION
**Pull request by cescoffier**
Blog post about the Mutiny variant of the Vert.x API.

CC @jponge

- [X] Do not merge without changing the date in the header. I've set it like that so you can read using surge. - DONE! Can be merged

### CODE REVIEW

**Comment by jponge** [comment]
Nice!

File: _posts/2020-11-24-mutiny-vertx.adoc [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+Several posts have already mentioned this, but it deserves a specific blog post.
+So, here we are.
+
+== Eclipse Vert.x
+
+https://vertx.io[Vert.x] is a toolkit to build reactive applications.
+The Vert.x ecosystem is enormous.
+From HTTP and data access abilities to messaging clients via microservice and security facilities, the Vert.x ecosystem is remarkably diverse and versatile.
+To understand that variety, just check the https://vertx.io/docs/[Vert.x documentation].
+That make Vert.x popular in many areas such as web applications, IoT gateways, banking applications and so on.
```
Comment (kenfinnigan) [inline _posts/2020-11-24-mutiny-vertx.adoc:22]: s/make/makes ?

File: _posts/2020-11-24-mutiny-vertx.adoc [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+== Where can we find this API?
+
+At the moment of writing, we only provide Vert.x core and Vert.x clients (MongoDB, Redis, Web client, Mqtt, and so on).
+We are extending the support to cover the full Vert.x stack.
+
+To use the Mutiny clients, you need to add the right dependency to your project.
+Browse https://search.maven.org/search?q=a:smallrye-mutiny-vertx*[the list of dependency] to pick the one you need.
+
+For example, you use the Mutiny variant of the Vert.x Web client, add the following dependency:
```
Comment (kenfinnigan) [inline _posts/2020-11-24-mutiny-vertx.adoc:193]: s/you/to ?

File: _posts/2020-11-24-mutiny-vertx.adoc [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+<dependency>
+  <groupId>io.smallrye.reactive</groupId>
+  <artifactId>smallrye-mutiny-vertx-web-client</artifactId>
+  <version>...</version>
+</dependency>
+----
+
+TIP: Something missing? Open an issue on https://github.com/smallrye/smallrye-reactive-utils/issues[SmallRye Reactive Utils].
+
+TIP: The Javadoc is available http://smallrye.io/smallrye-reactive-utils/apidocs/[there].
```
Comment (kenfinnigan) [inline _posts/2020-11-24-mutiny-vertx.adoc:206]: s/there/here ?

**Review by kenfinnigan — APPROVED** [review]
Couple of minor things only

**Pull request merged**